tests/MongoDB.Driver.Core.Tests/Core/Clusters/ClusterDescriptionTests.cs

```diff
@@ -53,10 +54,30 @@ static ClusterDescriptionTests()
         [Fact]
         public void CreateInitial_should_return_initial_description()
         {
-            var subject = ClusterDescription.CreateInitial(__clusterId, ClusterConnectionMode.Standalone);
+#pragma warning disable CS0618
+            var connectionModeSwitch = ConnectionModeSwitch.NotSet;
+            var subject = ClusterDescription.CreateInitial(__clusterId, connectionModeSwitch, ClusterConnectionMode.Standalone);
+#pragma warning restore CS0618
 
             subject.ClusterId.Should().Be(__clusterId);
+#pragma warning disable CS0618
             subject.ConnectionMode.Should().Be(ClusterConnectionMode.Standalone);
+#pragma warning restore CS0618
+            subject.DirectConnection.Should().Be(null);
+            subject.DnsMonitorException.Should().BeNull();
+            subject.LogicalSessionTimeout.Should().NotHaveValue();
+            subject.Servers.Should().BeEmpty();
+            subject.State.Should().Be(ClusterState.Disconnected);
+            subject.Type.Should().Be(ClusterType.Unknown);
+        }
+
+        [Fact]
+        public void CreateInitial_with_directConnection_should_return_initial_description()
+        {
+            var subject = ClusterDescription.CreateInitial(__clusterId, true);
+
+            subject.ClusterId.Should().Be(__clusterId);
+            subject.DirectConnection.Should().Be(true);
```

Comment:
Replace TWO CreateInitial test methods with ONE:

    #pragma warning disable 618
            [Theory]
            [InlineData(ConnectionModeSwitch.NotSet, ClusterConnectionMode.Automatic, null)]
            [InlineData(ConnectionModeSwitch.UseConnectionMode, ClusterConnectionMode.Automatic, null)]
            [InlineData(ConnectionModeSwitch.UseConnectionMode, ClusterConnectionMode.Direct, null)]
            [InlineData(ConnectionModeSwitch.UseDirectConnection, ClusterConnectionMode.Automatic, null)]
            [InlineData(ConnectionModeSwitch.UseDirectConnection, ClusterConnectionMode.Automatic, false)]
            [InlineData(ConnectionModeSwitch.UseDirectConnection, ClusterConnectionMode.Automatic, true)]
            public void CreateInitial_should_return_initial_description(ConnectionModeSwitch connectionModeSwitch, ClusterConnectionMode connectionMode, bool? directConnection)
            {
                var subject = ClusterDescription.CreateInitial(__clusterId, connectionMode, connectionModeSwitch, directConnection);

                subject.ClusterId.Should().Be(__clusterId);
                subject.ConnectionModeSwitch.Should().Be(connectionModeSwitch);
                switch (connectionModeSwitch)
                {
                    case ConnectionModeSwitch.UseConnectionMode: subject.ConnectionMode.Should().Be(connectionMode); break;
                    case ConnectionModeSwitch.UseDirectConnection: subject.DirectConnection.Should().Be(directConnection); break;
                }
                subject.DnsMonitorException.Should().BeNull();
                subject.LogicalSessionTimeout.Should().NotHaveValue();
                subject.Servers.Should().BeEmpty();
                subject.State.Should().Be(ClusterState.Disconnected);
                subject.Type.Should().Be(ClusterType.Unknown);
            }
    #pragma warning restore 618